xknx/knxip/device_configuration_request.py

```diff
@@ -0,0 +1,64 @@
+"""Module for Serialization and Deserialization of a KNX Device Configuration Request."""
+from __future__ import annotations
+
+from xknx.exceptions import CouldNotParseKNXIP
+
+from .body import KNXIPBody
+from .knxip_enum import KNXIPServiceType
+
+
+class DeviceConfigurationRequest(KNXIPBody):
+    """Representation of a KNX Device Configuration Request."""
+
+    SERVICE_TYPE = KNXIPServiceType.DEVICE_CONFIGURATION_REQUEST
```

Comment:
I think this (and the ACK) could share a common parent class with `TunnellingRequest`. They are exactly the same (when using `self.__class__.__name__` in `__repr__`).
But I'm fine with this as well, its just some duplicate code - we have a lot of that 😬

Reply:
How can a "private" parent class be created in python? But we could move that to a new PR.

Reply:
Yeah, a separate PR would be fine. 
I guess technically there is no "private class", but it can be prepended with `_` and not assign a `SERVICE_TYPE`. Maybe even not inherit from `KNXIPBody`, but use it as Mixin class.